form.yml.erb

```diff
@@ -26,10 +26,10 @@ attributes:
         label: "Method"
         help: |
             <ul>
-                <li><b>AlphaFold2.3</b> High Accuracy, Slower</li>
-                <li><b>ESMFold</b> Medium/Low Accuracy, Fastest, No Evolutionary Sequence Calculations</li>
-                <li><b>RoseTTAFold-All-Atom</b> High Accuracy, Slower</li>
-                <li><b>HelixFold3</b> High Accuracy, Slower</li>
+                <li><b>AlphaFold2.3</b> High Accuracy, Slower</li> - [Paper](https://doi.org/10.1038/s41586-021-03819-2)
```

Comment:
Links should be formatted using HTML, testing suggests markdown is not supported here
`<a href="https://doi.org/10.1038/s41586-021-03819-2">Paper</a>`

Reply:
Aha! Will do in the moment. Thought I read OoD supported Markdown but maybe not in this field

Reply:
I believe it is supported in some fields but here we should stick with HTML.